asaps/cli.py

```diff
@@ -1,12 +1,33 @@
+import datetime
+import logging
 import time
 
 from asnake.client import ASnakeClient
 import click
+import structlog
 
 from asaps import models
 
+logger = structlog.get_logger()
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H.%M.%S')
+log_file_name = f'log-{timestamp}.log'
+logging.basicConfig(format="%(message)s",
+                    handlers=[logging.FileHandler(log_file_name, 'w'),
+                              logging.StreamHandler()], level=logging.INFO)
```

Comment:
These 5 lines move inside the `main()` function. The only logging related thing that should be at the module level is
```python
logger = structlog.get_logger()
```